src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/TagBuilder.cs

```diff
@@ -144,6 +164,11 @@ public static string CreateSanitizedId(string name, string invalidCharReplacemen
             return stringBuffer.ToString();
         }
 
+        /// <summary>
+        /// Generates a sanitized ID attribute for the tag by using the specified name.
+        /// </summary>
+        /// <param name="name">The name to use to generate an ID attribute.</param>
+        /// <param name="idAttributeDotReplacement">A string to use in place of any dot characters in the name.</param>
```

Comment:
Actually uses `CreateSanitizedId` and that (as documented :smiley_cat:) checks for `valid HTML 4.01 "id" attribute` characters. We have a problem w/ the parameter name but let's not mislead the reader further.